Library/Homebrew/livecheck.rb

```diff
@@ -77,24 +77,48 @@ def strategy(symbol = nil)
   # Sets the `@url` instance variable to the provided argument or returns the
   # `@url` instance variable when no argument is provided. The argument can be
   # a `String` (a URL) or a supported `Symbol` corresponding to a URL in the
-  # formula (e.g., `:stable`, `:homepage`, or `:head`).
+  # formula/cask (e.g., `:stable`, `:homepage`, `:head`, `:cask_url`, `:appcast`).
   # @param val [String, Symbol] URL to check for version information
   # @return [String, nil]
   def url(val = nil)
     @url = case val
     when nil
       return @url
+    when :cask_url
+      @formula_or_cask.url.to_s
+    when :appcast
+      @formula_or_cask.appcast.to_s
```

Comment:
```suggestion
    when :appcast
      @formula_or_cask.appcast.to_s
    when :cask_url
      @formula_or_cask.url.to_s
```

This is simply stylistic, as these are vaguely in alphabetical order.